### ISSUE TITLE
Upgrade ember-cli-htmlbars to remove deprecation warning

### ISSUE THREAD

**Issue by jacobq** [heroku/ember-sortable#170]
The current release (1.11.0) triggers an annoying deprecation warning w/ recent versions of Ember that would be fixed by upgrading to the latest release of `ember-cli-htmlbars` (2.x). Please consider upgrading [this](https://github.com/jgwhite/ember-sortable/blob/b5e1d8aa8db302641e91178d8339fecd073f6ba1/package.json#L24).

```
├─┬ ember-sortable@1.11.0
│ └─┬ ember-cli-htmlbars@1.3.4
│   └── ember-cli-version-checker@1.3.1 
```

The deprecation warning looks like this and is described [here](https://github.com/ember-cli/ember-cli-version-checker/issues/48):
```
DEPRECATION: An addon is trying to access project.nodeModulesPath. This is not a reliable way to discover npm modules. Instead, consider doing: require("resolve").sync(something, { basedir: project.root }). Accessed from:   new NPMDependencyVersionChecker (/path/to/project/node_modules/ember-getowner-polyfill/node_modules/ember-cli-version-checker/src/npm-dependency-version-checker.js:11:33)
```

**Comment by Dhaulagiri** [comment]
@jacobq the warning you pasted references `ember-getownerpolyfill`, but I do see a similar warning in ember-sortable that references `ember-new-computed`.  I opened a PR to remove that in #175 which should address this (i think).

**Comment by jacobq** [comment]
@Dhaulagiri Thanks for looking into this! My understanding what the the old version (1.3.1) of ember-cli-version-checker would cause this all by itself. I will try to see if this clears the warning now.

**Comment by jacobq** [comment]
Hmm, I still see references to the old `ember-cli-version-checker` in v1.11.1 and #master, but when I set up a new/empty app and install the addon I don't get the warning now. Also, I can confirm that `ember-new-computed` appears to be gone now, as you said.

Latest release:
```
$ npm ls ember-cli-version-checker@1
test-app@0.0.0 /home/jquant/tmp/test-app
└─┬ ember-sortable@1.11.1
  ├─┬ ember-cli-htmlbars@1.3.4
  │ └── ember-cli-version-checker@1.3.1 
  └─┬ ember-new-computed@1.0.3
    └─┬ ember-cli-babel@5.2.4
      └── ember-cli-version-checker@1.3.1 
```

Current master:
```
$ ember install ember-sortable@jgwhite/ember-sortable.git#master
...
$ npm ls ember-cli-version-checker@1
test-app@0.0.0 /home/jquant/tmp/test-app
└─┬ ember-sortable@1.9.1 (github:jgwhite/ember-sortable#381809ccb35c51dad79ba78f71c30810977839b2)
  ├─┬ ember-cli-babel@5.2.4
  │ └── ember-cli-version-checker@1.3.1 
  └─┬ ember-cli-htmlbars@1.3.4
    └── ember-cli-version-checker@1.3.1  deduped
```

My PR, which attempts to upgrade to ember-cli@3.0.0:
```
$ npm ls ember-cli-version-checker@1
ember-sortable@1.11.1 /home/jquant/tmp/ember-sortable
└── (empty)
```
but it still has a few failing test :/

**Comment by jgwhite** [comment]
This project is now being maintained by the team at @heroku.

To get the project healthy and maintainable again, we're going to be upgrading ember-cli and all dependencies to their latest stable versions. Hopefully that will fix this issue. Watch this space.

**Comment by Dhaulagiri** [comment]
This should be fixed now in 1.12, please re-open if you notice otherwise
